### PR TITLE
Add option to enable multi-stream

### PR DESCRIPTION
**Pull request by DanielMcAssey**
Adds the option to enable multi-stream

### CODE REVIEW

**Review by saghul — CHANGES_REQUESTED** [review]

File: jvb/rootfs/defaults/jvb.conf [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{ $COLIBRI_REST_ENABLED := .Env.COLIBRI_REST_ENABLED | default "false" | toBool }}
 {{ $ENABLE_COLIBRI_WEBSOCKET := .Env.ENABLE_COLIBRI_WEBSOCKET | default "1" | toBool }}
 {{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool }}
+{{ $ENABLE_MULTI_STREAM := .Env.ENABLE_OCTO | default "0" | toBool }}
```
[the remark this answers is not in the document]
Comment (DanielMcAssey) [reply jvb/rootfs/defaults/jvb.conf]: Oops! Sorted

**Comment by DanielMcAssey** [comment]
Should be good to go now

File: web/rootfs/defaults/settings-config.js [hunk 2]
```diff
@@ -68,6 +69,12 @@ config.startWithVideoMuted = {{ $START_WITH_VIDEO_MUTED }};
 config.startBitrate = '{{ .Env.START_BITRATE }}';
 {{ end -}}
 
+{{ if $ENABLE_MULTI_STREAM -}}
+config.flags.sourceNameSignaling = true;
```
[the remark this answers is not in the document]
Comment (DanielMcAssey) [reply web/rootfs/defaults/settings-config.js]: Sorted :)

**Review by saghul — APPROVED** [review]
LGTM!

**Pull request merged**